modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -478,7 +478,10 @@ export default {
       this.isLoading = false;
     },
 
-    async onContentChanged({ action }) {
+    async onContentChanged({ action, doc }) {
+      if (doc.type !== '@apostrophecms/image' || ![ 'archive', 'update' ].includes(action)) {
```

Comment:
Right now the media manager is only used for image pieces. If it was ever used for some other piece type it would be on purpose. So I don't know why the piece type needs to appear here?

Reply:
`content-changed` bus event is emitted for other pieces too.. Like for images tags when editing an image.